dev-docs/source/GettingStarted/GettingStarted.rst

```diff
@@ -46,7 +53,9 @@ If you need further help, ask another developer at the facility how to configure
 
 Building Mantid
 ###############
-See :ref:`BuildingWithCMake` for information about building Mantid.
+
+| From this point, you can proceed to build Mantid. See :ref:`BuildingWithCMake` for information.
+| For developers at ISIS, continue reading for information on accessing the archive.
```

Comment:
Has this been indented by accident?

Reply:
It shouldn't be indented, I *think* this just forces a new line without adding paragraph spacing. Totally optional, just helps readability imo